rosa_learning/creating_cluster_workshop/cloud-experts-getting-started-support.adoc

```diff
@@ -0,0 +1,70 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-support"]
+= Tutorial: Obtaining support
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-support
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-01-17
+
+Finding the right help when you need it is important. These are some of the resources at your disposal when you need assistance.
+
+== Adding support contacts
+You can add additional email addresses for communications about your cluster. 
+
+. On the {cluster-manager-first} user interface (UI), click *Cluster List* in the side navigation tabs.
+. Select the cluster that needs support.
+. Click the *Support* tab.
+. Click *Add notification contact*, and enter the additional email addresses.
+
+== Contacting Red{nbsp}Hat for support using the UI
+
+. On the {cluster-manager} UI, click the *Support* tab.
+. Click *Open support case*.
+
+== Contacting Red{nbsp}Hat for support using the support page
+
+. Go to the link:https://support.redhat.com[Red{nbsp}Hat support page].
+. Click *Open a new Case*.
++
+image::obtain-support-case.png[]
+
+. Log in to your Red{nbsp}Hat account.
+. Select the reason for contacting support.
++
+image::obtain-support-reason.png[]
+
+. Select *Red{nbsp}Hat OpenShift Service on AWS*.
+
```

Comment:
You need a `+` on this line to get the image and numbers to order correctly

![image](https://github.com/user-attachments/assets/6191dde5-4699-418d-9b66-80d94a671d98)